usl_models/tests/flood_ml/flood_model_test.py

```diff
@@ -262,3 +146,16 @@ def test_model_checkpoint():
     new_weights = new_model._model.get_weights()
     for old, new in zip(old_weights, new_weights):
         np.testing.assert_array_equal(old, new)
+
+
+def test_get_temporal_window():
+    """Tests copmuting the temporal window."""
+    B, T_MAX, M = 1, 16, 6
+    temporal = tf.random.normal((B, T_MAX, M))
+
+    t = 1  # Timestep
+    n = 5  # Window size
+    actual = flood_model.FloodConvLSTM._get_temporal_window(temporal, t=t, n=n)
+    assert actual.shape == (B, n, M)
+    expected = tf.concat([np.zeros((B, n - t, M)), temporal[:, 0:t]], axis=1)
```

Comment:
This basically seems to just replicate the code from the actual method. Maybe it would be useful to create observable tensors? This has the benefit of also helping users understand what exactly the method is doing.

I'm picturing something similar to https://github.com/UrbanSystemsLab/climateiq-cnn/blob/main/usl_models/tests/atmo_ml/atmo_utils_test.py.

Reply:
Done.